maintainers/maintainer-list.nix

```diff
@@ -18526,6 +18526,12 @@
     githubId = 1486805;
     name = "Toon Nolten";
   };
+  tornax = {
```

Comment:
**Flip**

Reply:
The commit adding you as new maintainer should be the first on the stack.
Also, it should contain only this.

Reply:
done